sql/src/main/kotlin/io/github/codebandits/beak/DataAccessError.kt

```diff
@@ -7,4 +7,8 @@ sealed class DataAccessError {
         data class ConnectionError(override val cause: Throwable) : SystemError()
         data class TransactionError(override val cause: Throwable) : SystemError()
     }
+
+    sealed class EntityError: DataAccessError() {
```

Comment:
I'm thinking of renaming this QueryError, OperationError, or something else. I haven't figured out what I like yet so I haven't renamed it yet.